app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,76 @@
+class TweetController < ApplicationController
+
+	get '/tweets/new' do 
+		if logged_in?
+		erb :'/tweets/create_tweet'
+	else
+		redirect to '/login'
+	end
+	end
+
+  post '/tweets' do
+  	if params[:content] != ""
+	  	@tweet = Tweet.new(params)
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(params)
```
Both approaches are good, but in some cases you may find this option useful.